content/pt-br/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -0,0 +1,586 @@
+
+{{< codenew file="pods/probe/grpc-liveness.yaml" >}}
+
+Para usar a verificação gRPC, uma `porta` precisa ser configurada. Se você quiser distinguir verificações de diferentes tipos e recursos você pode usar o campo `service`.
+Você pode atribuir `service` ao valor `liveness` e fazer o seu `endpoint de Health Checking gRPC` responder a esta requisição diferentemente de quando ao campo `service` estiver atribuído `readiness`. 
+Isto permite você usar o mesmo `endpoint` para diferentes tipos de verificações de integridade de contêiner (diferente de necessitar escutar em duas portas diferentes).
+Se você precisar especificar seu nome de serviço personalizado e também especificar um tipo de verificação, o projeto kubernetes recomenda a você usar um nome que os concatene. Por exemplo: `myservice-liveness` (usando `-` como um separador).  
+
+{{< note >}}
+Diferentemente de verificações HTTP e TCP, portas nomeadas não podem ser usadas 
+e host personalizado não pode ser configurado.
+{{< /note >}}
+
+Problemas de configuração (por exemplo: porta e serviço incorretos, 
+protocolo de verificação de integridade não implementado) são considerados 
+falhas de verificação, similarmente às verfificações HTTP e TCP.
+
+Para testar verificações de _liveness_ gRPC, crie um Pod usando o seguinte comando.
+No exemplo a seguir, o pod etcd está configurado para usar _liveness probe_ gRPC.
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/probe/grpc-liveness.yaml
+```
+
+Após 15 segundos, veja os eventos do Pod para verificar que o teste de _liveness_ 
+não falhou:
+
+```shell
+kubectl describe pod etcd-with-grpc
+```
+
+Antes da versão 1.23 do Kubernetes, verificações de integridade gRPC foram frequentemente 
+implementadas usando [grpc-health-probe](https://github.com/grpc-ecosystem/grpc-health-probe/),
+como descrito no post do blog [verificação de integridade gRPC em servidores no Kubernetes]
+(/blog/2018/10/01/health-checking-grpc-servers-on-kubernetes/).
+O comportamento de verificação gRPC interno é semelhante 
+ao implementado pelo `grpc-healthh-probe`.
+Ao migrar de `grpc-health-probe` para verificações incorporadas, lembre-se 
+das seguintes diferenças:
+- Verificações incorporadas executam contra o endereço IP do pod, 
+  diferente do `grpc-health-probe` que executa no ip local `127.0.0.1`.
+  Certifique-se de configurar seu `endpoint` gRPC para escutar no endereço IP do Pod.
+- As verificações incorporadas não suportam nenhum parâmetro de autenticação (como `-tls`).
+- Não há códigos de erro para verficações incorporadas. 
+  Todos os erros são considerados falhas de verfificação.
+- Se o recurso `ExecProbeTimeout` estiver definido como `false`, grpc-health-probe 
+  **não** respeitará a configuração `timeoutSeconds` (que por padrão é de 1s),
+  enquanto a verficação embutida falharia com um `timeout`.
+
+## Use uma porta nomeada
+
+Você pode usar [`portas`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#ports)
```

Comment:
```suggestion
Você pode usar [portas](/docs/reference/kubernetes-api/workload-resources/pod-v1/#ports)
```

Although this was formatted as code before, it doesn't look correct as such in the translated version.